package.json

```diff
@@ -34,31 +34,34 @@
     "docs": "docs-ts"
   },
   "peerDependencies": {
-    "fp-ts": "^2.7.1"
+    "fp-ts": "^2.10.5",
+    "monocle-ts": "^2.3.10"
```

Comment:
In thinking about this PR further, the addition of `monocle-ts` as a peer-dependency of `expressive-ts` would be considered a breaking change for users of the library.  The update to the `fp-ts` version is also a soft breaking change, since it's a peer dependency of the lib.

I would like to avoid implementing any breaking changes until the `fp-ts@3.0.0` release, at which point we can collaborate on a revamp of the entire `expressive-ts` library.

We can leave this PR open until then if that is alright?

Reply:
Sounds good